joomla/joomla-cms
# [com_config] add js message titles

#### Summary of Changes

This simple PR adds the js message titles in com_config (application and component view).
#### Testing Instructions
1. Use latest staging
2. Go to Configuration and send a test mail (or any behaviour that returns a js message - ex: permissions). Result: JS message WITHOUT the title. 
   ![image](https://cloud.githubusercontent.com/assets/9630530/15926891/3c3cbf8a-2e36-11e6-91f4-9d05e2fe53a9.png)
3. Apply patch
4. Repeat step 1. (remember to clear browser cache in config page). Result: JS message WITH the title. 
   ![image](https://cloud.githubusercontent.com/assets/9630530/15926851/07a0cdfc-2e36-11e6-9b29-e4a5ba278fe3.png)

This should work for all message, notice, warning and error messages in com_config and component config.
